packages/@react-aria/overlays/src/Overlay.tsx

```diff
@@ -23,7 +23,9 @@ export interface OverlayProps {
    */
   portalContainer?: Element,
   /** The overlay to render in the portal. */
-  children: ReactNode
+  children: ReactNode,
+  /** Removes our focus scope, you are expected to manage the focus instead. */
```

Comment:
Let's not write comments/docs in the first person.

```suggestion
  /** 
   * Disables default focus management for the overlay, including containment and restoration.
   * This option should be used very carefully. When focus management is disabled, you must 
   * implement focus containment and restoration yourself to ensure the overlay is keyboard accessible.
   */
```